syntax = "proto3";

package Metrics;

service QueryManager {
    rpc QueryMetrics (MetricsRequest) returns (MetricsResponse) {};
}

message MetricsRequest{
    string metrics = 1;
}

message MetricsResponse{
    string data = 1;
}